README.md

```diff
@@ -10,10 +10,13 @@
 The Audit scanner inspects the resources defined in the cluster and
 identifies the ones that are violating Kubewarden policies.
 
-The results of the scan are made available via `PolicyReport` objects. Each Namespace
+The results of the scan can be made available via `PolicyReport` objects. Each Namespace
 has its own dedicated `PolicyReport`. Cluster-wide resources compliance is available via
 the `ClusterPolicyReport` resource. 
 
+Instead of relying on `PolicyReport` objects, one can also configure Audit scanner to
+save all this information in-memory only, by specifying `--store MEMORY`.
+
 # Deployment
```

Comment:
(opening thread here randomly)

> One thing that we can also do in the future is to parse (in some way) the toJSON() result and instead of logging the entire report in a single log block, we could have a log line per violation. What do you think?

At that point, I think it would be better  to expose the reports with proper structured logging.

Reply:
yeap, completely agree with that. but perhaps it can be another feature request with another PR, or do you think it's feasible to include that implementation under this one?

Reply:
I too think it would be better to get this done with another PR